### PR TITLE
[Rector] Apply Rector: ForToForeachRector

### PR DESCRIPTION
**Pull request by samsonasik**
Apply `ForToForeachRector` rector rule to use foreach instead of for when possible.

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

File: system/Helpers/url_helper.php [hunk 1]
```diff
@@ -447,9 +447,9 @@ function safe_mailto(string $email, string $title = '', $attributes = ''): strin
 		$output = '<script type="text/javascript">'
 				. 'var l=new Array();';
 
-		for ($i = 0, $c = count($x); $i < $c; $i ++)
+		foreach ($x as $i => $x)
```
Comment (paulbalandan) [inline system/Helpers/url_helper.php:450]: Is this not good practice, overwriting the original?

Comment (samsonasik) [reply system/Helpers/url_helper.php:450]: The singularizing process only works in rector if the loop variable is a plural. I can manually change if that needed. Actually, the `$x` still works. Do you have suggestion of new variable of value var ?

Comment (paulbalandan) [reply system/Helpers/url_helper.php:450]: Ok. I always thought this type of assignment will overwrite the original. But maybe we can use another letter for the assignment, to make it less confusing.

Comment (samsonasik) [reply system/Helpers/url_helper.php]: I updated to `$value`

**Review by paulbalandan — APPROVED** [review]

**Comment by samsonasik** [comment]
Thank you @paulbalandan  for the review.

**Pull request merged**